File: Python/Medium/to_scottish_screaming.py
# https://edabit.com/challenge/2hvruws6kgiKj98Rvs
def to_scottish_screaming(txt):
    txt=txt.upper()
    return "".join(letter if letter not in 'AIOU' else "E" for letter in list(txt))

print(to_scottish_screaming("lorem ipsum"),"LEREM EPSEM")
print(to_scottish_screaming("Leeroy jenkins!"),"LEEREY JENKENS!")
print(to_scottish_screaming("A, wonderful, day, don't, you, think?"),"E, WENDERFEL, DEY, DEN'T, YEE, THENK?")
print(to_scottish_screaming("Hello world"),"HELLE WERLD")
print(to_scottish_screaming("start queueing you oafs"),"STERT QEEEEENG YEE EEFS")

File: javascript/Very_Easy/toScottishScreaming.js
// https://edabit.com/challenge/eRuGNqijDdFpLsW5K

function toScottishScreaming(str) {
    const vowels = {'a':'e','i':'e','o':'e','u':'e'};
	const nextString= str.toLowerCase().replace(/[aiou]/g,l=>vowels[l]).toUpperCase()
    return nextString
}



console.log(toScottishScreaming("lorem ipsum"),"LEREM EPSEM")
console.log(toScottishScreaming("Leeroy jenkins!"),"LEEREY JENKENS!")
console.log(toScottishScreaming("A, wonderful, day, don't, you, think?"),"E, WENDERFEL, DEY, DEN'T, YEE, THENK?")
console.log(toScottishScreaming("Hello world"),"HELLE WERLD")
console.log(toScottishScreaming("start queueing you oafs"),"STERT QEEEEENG YEE EEFS")